crowdkit/aggregation/image_segmentation/segmentation_rasa.py

```diff
@@ -14,20 +14,26 @@
 
 @attr.s
 class SegmentationRASA(BaseImageSegmentationAggregator):
-    """Segmentation RASA - chooses a pixel if sum of weighted votes of each workers' more than 0.5.
+    r"""The **Segmentation RASA** (Reliability Aware Sequence Aggregation) algorithm chooses a pixel if the sum of the weighted votes of each worker is more than 0.5.
 
-    Algorithm works iteratively, at each step, the workers are reweighted in proportion to their distances
-    to the current answer estimation. The distance is considered as $1 - IOU$. Modification of the RASA method
-    for texts.
+    The Segmentation RASA algorithm consists of three steps:
+    1. performs the weighted Majority Vote algorithm;
```

Comment:
Let's replace these phrases with complete sentences (`1. Performs the weighted Majority Vote algorithm.` and so on)